public/static/locales/de/donate.json

```diff
@@ -58,5 +58,6 @@
   "noProjectsFound": "Keine Projekte gefunden",
   "inTimeOfTaxReturns": "rechtzeitig für deine Steuererklärung.",
   "titleToShare": "Bäume pflanzen gegen die Klimakrise!",
-  "textToShare": "Um die Klimakrise zu verhindern, müssen die Kohlenstoffemissionen drastisch reduziert und Bäume gepflanzt werden. Deshalb habe ich gerade einige gepflanzt. Schau unter {{linkToShare}} nach, wenn du auch welche pflanzen willst! #StopTalkingStartPlanting #TrillionTrees @PftP_int"
+  "textToShare": "Um die Klimakrise zu verhindern, müssen die Kohlenstoffemissionen drastisch reduziert und Bäume gepflanzt werden. Deshalb habe ich gerade einige gepflanzt. Schau unter {{linkToShare}} nach, wenn du auch welche pflanzen willst! #StopTalkingStartPlanting #TrillionTrees @PftP_int",
+  "thankyouHeaderText": "Wir sind auf dem Weg zu einer \n <h1>Billion Bäume</h1>"
```

Comment:
Text is fine. Will be reviewed and retranslated in LingoHub anyway!

Reply:
Yes it was for testing purpose